external/fv3fit/fv3fit/keras/_models/shared/callbacks.py

```diff
@@ -2,8 +2,21 @@
 import tensorflow as tf
 import wandb
 
+# Add third party callbacks here to make them available in training
+ADDITIONAL_CALLBACKS = {"WandbCallback": wandb.keras.WandbCallback}
```

Comment:
Why was this renamed? Our callbacks are also third party callbacks (from the standpoint of Keras), and the error message describing these as "third party callbacks" wasn't updated. I'd suggest either updating that error message or reverting the variable name change.

Reply:
Sure. I thought 'third party' mean third party relative to the us+keras.

Reply:
That's not an unreasonable interpretation, renaming these could be fine.